tvbwidgets/tests/test_pse_stage_in.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+#
+# "TheVirtualBrain - Widgets" package
+#
+# (c) 2022-2023, TVB Widgets Team
+#
+
+import os
+import pytest
+import numpy as np
+from tvb.datatypes import connectivity
+from tvb.simulator.coupling import Kuramoto
+from tvb.simulator.models.epileptor import Epileptor
+from tvb.simulator.simulator import Simulator
+from tvbwidgets.core.pse.toml_storage import TOMLStorage
+
+
+@pytest.fixture
+def custom_simulator():
+    """ Returns a customized Simulator """
+    simulator = Simulator(connectivity=connectivity.Connectivity.from_file())
+    simulator.coupling = Kuramoto()
+    simulator.conduction_speed = 3.5
+    simulator.simulation_length = 1100.0
+    simulator.model = Epileptor(a=np.array([2.0]), b=np.array([4.0]))
+    simulator.model.variables_of_interest = ['x1', 'x2', 'x2 - x1']
+    simulator.model.state_variable_range['x1'] = np.array([-4., 1.])
+
+    return simulator
+
+
+def test_stage_in_params(custom_simulator):
+    param1 = "conduction_speed"
```

Comment:
Add unit-test when serializing a PSE where param1 and then param2 is a `connectivity`